### PR TITLE
Portfolio: mobile version: Works section

### PR DESCRIPTION
**Pull request by principles31**
Portfolio: mobile version: Works section 
Works Section was created, 
this project contains
- index.html
- works and index CSS
- Additional images
- linters were enabled to check for bugs

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -41,12 +41,114 @@ <h1>Emmanuel Mensah</h1>
...
+                    <div class="grid_1 grid_picture"></div>
+                    <div class="grid_row">
+                        <p class="pro_text text_body_font">Tonic</p>
+                        <ul class="list_category text_body_font">
+                            <li>Html</li>
+                            <li>CSS</li>
+                            <li>Javascript</li>
+                        </ul>
+                        <div class="project_click_class text_body_font">
+                           <p> See this project </p>
```
Comment (Cathella) [inline index.html]: - [x] Kindly make `See this project` a link a or button tag, and follow the button styling: Active, Hover and Pressed (as seen in the design)

File: index.html [hunk 2]
```diff
@@ -41,12 +41,114 @@ <h1>Emmanuel Mensah</h1>
...
+                        </ul>
+                        <div class="project_click_class text_body_font">
+                            <p> See this project </p>
+                            <div class="arrow_key"></div>
+                         </div>
+                    </div>
+
+
+                </div>
+            </section>
```
Comment (Cathella) [inline index.html]: - [x] Please go on and include the about section as it is a requirement in this milestone 🙏🏻.

**Review by Cathella — CHANGES_REQUESTED** [review]
## 🟡 Status: Changes Required 🛑

### Hi @principles31 👋🏻, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## To Highlight 👏🏻

✅ No linter errors.
✅ You used correct Github Flow.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (principles31) [reply index.html]: Thanks, @Cathella, it will be done.

File: index.html (hunk 1 above)
Comment (principles31) [reply index.html]: @Cathella,please kindly assist in pressed CSS styling

Comment (Cathella) [reply index.html]: I think you can just use the current styles and apply them to a button or link instead

File: index.html (hunk 2 above)
Comment (principles31) [reply index.html]: @Cathella https://github.com/principles31/portfolio/pull/3 CHeck this pull request with the about page added.

**Comment by principles31** [comment]
@Cathella https://github.com/principles31/portfolio/pull/3 CHeck this pull request with the about page added.

**Pull request merged**